e2e/api/steps.js

```diff
@@ -310,55 +310,7 @@ module.exports = {
 
     console.log('General application case state : APPLICATION_SUBMITTED_AWAITING_JUDICIAL_DECISION');
     console.log('*** GA Case Reference: ' + gaCaseReference + ' ***');
-
-    return gaCaseReference;
-  },
-
-  initiateGeneralApplicationWithOutNoticeUncloaked: async (user, parentCaseId) => {
```

Comment:
Are we not using this method initiateGeneralApplicationWithOutNoticeUncloaked anywhere in tests

Reply:
no we are not @deepthidoppalapudihmcts hence i removed